app/components/Countdown.tsx

```diff
@@ -14,47 +14,53 @@ interface ICountdownComponentProps {
 export const CountdownComponent = ({ duration }: ICountdownComponentProps) =>
   <div className={styles.countdown}>
     <strong className={styles.label}>d</strong>
-    <span className={styles.value}>
+    <span className={styles.value} data-test-id="countdown-days">
```

Comment:
Would you leave this in even in production release?

Reply:
Yup. Currently, it stays like this. In future, we can experiment with using babel to remove these tags. Something like that: https://www.npmjs.com/package/babel-plugin-jsx-remove-qa